ocrd_tesserocr/ocrd-tool.json

```diff
@@ -33,7 +33,7 @@
     "ocrd-tesserocr-recognize": {
       "executable": "ocrd-tesserocr-recognize",
       "categories": ["Text recognition and optimization"],
-      "description": "Recognize text in lines with tesseract",
+      "description": "Recognize text lines with Tesseract (using annotated derived images, or masking and cropping images from coordinate polygons)",
```

Comment:
For me, *text in lines* is less ambig than *text lines* which may be misunderstood as detecting lines.

Reply:
True, you might read that as "line segmentation"

Reply:
cf 
8064c0a